pkg/s3wrapper/client.go

```diff
@@ -54,10 +54,11 @@ type API interface {
 	UpdateObjectTimestamp(ctx context.Context, objectName string) (bool, error)
 	ExpireObjects(ctx context.Context, prefix string, deleteTime time.Duration, callback func(ctx context.Context, log logrus.FieldLogger, objectName string))
 	ListObjectsByPrefix(ctx context.Context, prefix string) ([]string, error)
-	UploadBootFiles(ctx context.Context) error
-	DoAllBootFilesExist(ctx context.Context) (bool, error)
-	DownloadBootFile(ctx context.Context, fileType string) (io.ReadCloser, string, int64, error)
-	GetS3BootFileURL(fileType string) string
+	UploadBootFiles(ctx context.Context, openshiftVersion string) error
+	DoAllBootFilesExist(ctx context.Context, isoObjectName string) (bool, error)
+	DownloadBootFile(ctx context.Context, isoObjectName, fileType string) (io.ReadCloser, string, int64, error)
+	GetS3BootFileURL(isoObjectName, fileType string) string
```

Comment:
Maybe it would be nicer to have openshiftVersion as the parameter instead of isoObjectName?

Reply:
I think it's much flexible to have the ISO received as a parameter, as been done for the rest of the boot functions.